### ISSUE TITLE
Add support for newer sensors

### ISSUE THREAD

**Issue by danhausman** [vojtamolda/homebridge-ecobee3-sensors#41]
Would it be possible to add support for newer sensors?
[11/27/2022, 11:20:17 AM] [Ecobee 3 Sensors] [My ecobee] Not-supported measurement type | airQualityAccuracy
[11/27/2022, 11:20:17 AM] [Ecobee 3 Sensors] [My ecobee] Not-supported measurement type | airQuality
[11/27/2022, 11:20:17 AM] [Ecobee 3 Sensors] [My ecobee] Not-supported measurement type | vocPPM
[11/27/2022, 11:20:17 AM] [Ecobee 3 Sensors] [My ecobee] Not-supported measurement type | co2PPM
[11/27/2022, 11:20:17 AM] [Ecobee 3 Sensors] [My ecobee] Not-supported measurement type | airPressure
[11/27/2022, 11:20:17 AM] [Ecobee 3 Sensors] [My ecobee] Initialized | 531657384941

**Comment by vojtamolda** [comment]
Yeah. Support for these would certainly be nice. I'm sorry but I don't have the bandwidth to implement these right now. But I'll be happy to merge a PR!
